draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -651,29 +622,29 @@
    ----------------+
 					</artwork>
 				</figure>
-				<t>The algorithm starts with cwin := LPP + 1. In the congestion avoidance phase, cwin increases as LSPs are acked: for every acked LSP, cwin += 1 / cwin. Thus, the sending rate roughly increases linearly with the RTT. Since the RTT is low in many IS-IS deployments, the sending rate can reach fast rates in short periods of time.</t>
+				<t>The algorithm starts with cwin sets to cwin = LPP + 1. In the congestion avoidance phase, cwin increases as LSPs are acked: for every acked LSP, cwin += 1 / cwin. Thus, the sending rate roughly increases linearly as the RTT decrease. Since the RTT is low in many IS-IS deployments, the sending rate can reach fast rates in short periods of time.</t>

```

Comment:
s/starts with cwin sets to cwin/starts with cwin

(the intended change is only to change ":=" to "=")

s/RTT decrease/RTT decreases

Reply:
Proposed alternative:
"Thus, the sending rate is inversely proportional to the RTT"

Reply:
@lesginsberg thanks. Done. 
@gsol10 thanks. Done. (a little more complex as per Acee's feedback, but more detailed from a math standpoint.